rafthttp/http.go

```diff
@@ -72,6 +72,18 @@ func (h *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if err := checkPeerVersionCompatibility(serverVersion(r.Header), minClusterVersion(r.Header)); err != nil {
+		log.Printf("rafthttp: request ignored due to %v", err)
+		// StatusForbidden has been used to forbid messages from removed member,
```

Comment:
hmm... i think we should still use statusForbidden... the other side can check the string. we need to define a const string error though.